### DISCUSSION TITLE
Reaction building block for large molecule model

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1430] [Questions & Problems] [answered]
I am wondering if someone can explain exactly how the reactions in the large molecule model are implemented. For example, I can see that the molecule "FcRn kinetics endosome" is used as a modifier of the reaction "FcRn binding Aciclovir in endosomal space", but it doesn't actually appear in the formula so I'm not sure what role this molecule is playing. Also, according to the MSV, "FcRn kinetics endosome" is only present in the Organism/EndogenousIgG/Endosome compartment, but the start value is set to 0, so I don't see if we actually have any of this molecule present. So what is this molecule doing?

![image](https://user-images.githubusercontent.com/57540473/210999649-a381e7be-4d79-4d63-9865-b51ca62ce8d6.png)

**Comment by StephanSchaller** [comment, marked as answer]
This is a functionality/necessity from earlier Versions of MoBi, where Molecules were used as "booleans" to determine localization of reactions. This could now also be solved with the "Container Criteria" of the Reaction. Shout set this as a feature whish...

**Reply by moPellowe** [reply]
Okay, that makes sense. So the molecules "FcRn kinetics *" could all be replaced with appropriate container criteria. Thank you for the explanation.
